CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Various fixes [Theodor]
```

Comment:
You say above that INITRAMFS_TASK = "" is introduced by the Tegra BSP. So not really just fixes for things that never surfaced until know